lib/go-tc/users.go

```diff
@@ -294,6 +294,7 @@ type UserV40 struct {
 	Tenant               *string    `json:"tenant"`
 	TenantID             int        `json:"tenantId" db:"tenant_id"`
 	Token                *string    `json:"-" db:"token"`
+	UCDN                 *string    `json:"ucdn"`
```

Comment:
It looks like if this is `NULL` it'll be coerced to an empty string in JWTs - does it need to be "null-able"?